Rubberduck.Core/Properties/AssemblyInfo.cs

```diff
@@ -31,4 +31,4 @@
 //      Buildnummer
 //      Revision
 //
-[assembly: AssemblyVersion("2.1.*")]
+[assembly: AssemblyVersion("2.2.*")]
```

Comment:
IIRC there was code that relied on the AssemblyVersions for Core and Main to match. Also note that the COM-facing dll is Rubberduck.Main, so if we only upgrade a single dll, it *should* be `Main` ...

Reply:
See this: https://github.com/bclothier/Rubberduck/blob/44f78870971a9dd4fcd985821a256d549e7faf2a/Rubberduck.Core/UnitTesting/ProjectTestExtensions.cs#L15 (Introduced in #3888)